packages/core/src/controllers/OptionsController.ts

```diff
@@ -182,6 +187,10 @@ export const OptionsController = {
     Object.assign(state, options)
   },
 
+  setBasic(basic: boolean) {
+    state.basic = basic
```

Comment:
Can we give this a more specific name rather than `basic`? Also to support our ESLint rule for namings as well, like `isAppKitBasic`

+ it should be compatibile for future feature additions as well; `isAppKitHeadless`

Or we can store a `appKitType` which have values like `"full" | "basic" | "headless"`

Reply:
removed this, we are not really storing it anymore. we handle this with class split and path exports